### ISSUE TITLE
Adding a role is confusing 

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#6920]
Many candidates failed to add role when creating scope for multiple attempts. Even though “Enter a valid role and press `Enter`” is shown in helper text and placeholder, still users fail to notify it. Better if we can use autocomplete select for the role selection, if it is not a major security concern.

![image](https://user-images.githubusercontent.com/20179540/69424557-f0654400-0d4e-11ea-92b6-96718fab901d.png)

**Comment by bhathiya** [comment]
I don't think auto-complete is a good idea security-wise. We'll have to figure out something else to solve this problem. Maybe just ask them to add a comma-separated list. 

or we can do it stackoverflow way. seperate by space. if there are no spaces when save, just use that as a single role.

**Comment by erangatl** [comment]
> or we can do it stackoverflow way. seperate by space. if there are no spaces when save, just use that as a single role.

+1 to automatically add tags on space, comma or semicolon

**Comment by dushaniw** [comment]
Will fix this for next 3.x release
